Can't login to Amazon Fire HD Fully Kiosk Browser

### Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Hi,

After update to 2026.1.0b0 I first beta can not login to my dashboard on Amazon Fire HD tab. I tried to clear cache but nothing changed. I can not enter username and password also on the screen. Only this screen is visible. I click Log In, it becames grey, waiting a bit and it turns to blue again, bot processing to the main dashboard.

<img width="179" height="171" alt="Image" src="https://github.com/user-attachments/assets/2db10104-ebca-4e6d-b91e-ac4b85d8c94d" />

### Describe the behavior you expected

After I click Log In, it should open my main dashboard.

### Steps to reproduce the issue

1.Restart the tab.
2.Cleared the cache.
3.
...


### What version of Home Assistant Core has the issue?

2026.1.0b0

### What was the last working version of Home Assistant Core?

2026.12.4

### In which browser are you experiencing the issue?

Fully Kiosk Browser

### Which operating system are you using to run this browser?

Amazon Fire HD

### State of relevant entities

```txt

```

### Problem-relevant frontend configuration

```yaml

```

### JavaScript errors shown in your browser console/inspector

```txt

```

### Additional information

_No response_

> If anybody is able to provide errors, please do. 🙏 Also kindly provide if you can load demo.home-assistant.io, if no error can be retrieved.

Here is. 

In Google Chrome it loads fine but It fails to load in Fully Kiosk Browser, but It was working fine in 2025.12.x

```
View the Webview Console output in real time

10:00:53.603 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:54.013 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:54.391 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:54.782 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:55.135 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:55.544 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:55.959 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:56.325 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:56.723 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:00:57.170 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of http://10.0.0.11:8123/frontend_es5/app.e8bc249594efe997.js

10:01:23.455 -- [ERROR] Failed to subscribe to system data, setting to empty object -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:24.093 -- [ERROR] Uncaught (in promise) #<Object> -- on line 0 of https://demo.home-assistant.io/#/lovelace

10:01:25.056 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.287 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.287 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.287 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.298 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.298 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.298 -- [ERROR] Uncaught (in promise) ReferenceError: _k is not defined -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:25.342 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.879 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.879 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.879 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.879 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.880 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.880 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js

10:01:28.880 -- [LOG] unmocked callService system_log write [object Object] -- on line 2 of https://demo.home-assistant.io/frontend_es5/main.488d84cc764fb803.js
```

<img width="2000" height="1200" alt="Image" src="https://github.com/user-attachments/assets/cb5faf41-1733-406b-820e-1acd666a521a" />

--- a/src/auth/ha-auth-form-string.ts
+++ b/src/auth/ha-auth-form-string.ts
@@ -38,13 +38,11 @@ export class HaAuthFormString extends HaFormString {
         }
       </style>
       <ha-auth-textfield
-      .type=${
-        !this.isPassword
+        .type=${!this.isPassword
           ? this.stringType
           : this.unmaskedPassword
             ? "text"
-            : "password"
-      }
+            : "password"}
         .label=${this.label}
         .value=${this.data || ""}
         .helper=${this.helper}
@@ -55,18 +53,17 @@ export class HaAuthFormString extends HaFormString {
         .name=${this.schema.name}
         .autocomplete=${this.schema.autocomplete}
         ?autofocus=${this.schema.autofocus}
-        .suffix=${
-          this.isPassword
-            ? // reserve some space for the icon.
-              html`<div style="width: 24px"></div>`
-            : this.schema.description?.suffix
-        }
-        .validationMessage=${this.schema.required ? this.localize?.("ui.panel.page-authorize.form.error_required") : undefined}
+        .suffix=${this.isPassword
+          ? // reserve some space for the icon.
+            html`<div style="width: 24px"></div>`
+          : this.schema.description?.suffix}
+        .validationMessage=${this.schema.required
+          ? this.localize?.("ui.panel.page-authorize.form.error_required")
+          : undefined}
         @input=${this._valueChanged}
         @change=${this._valueChanged}
-        ></ha-auth-textfield>
-        ${this.renderIcon()}
-      </ha-auth-textfield>
+      ></ha-auth-textfield>
+      ${this.renderIcon()}
     `;
   }
 }

